### PR TITLE
chore: adding priors to coloc step

### PR DESCRIPTION
**Pull request by addramir**
## 🚦 Before submitting

- [ ] Do these changes cover one single feature (one change at a time)?
- [ ] Did you read the [contributor guideline](https://opentargets.github.io/gentropy/development/contributing/#contributing-checklist)?
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you make sure there is no commented out code in this PR?
- [ ] Did you follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standards in PR title and commit messages?
- [ ] Did you make sure the branch is up-to-date with the `dev` branch?
- [ ] Did you write any new necessary tests?
- [ ] Did you make sure the changes pass local tests (`make test`)?
- [ ] Did you make sure the changes pass pre-commit rules (e.g `poetry run pre-commit run --all-files`)?

### CODE REVIEW

File: src/gentropy/colocalisation.py [hunk 1]
```diff
@@ -32,6 +35,9 @@ def __init__(
             credible_set_path (str): Input credible sets path.
             coloc_path (str): Output Colocalisation path.
             colocalisation_method (str): Colocalisation method.
+            priorc1 (float, optional): Prior on variant being causal for trait 1. Defaults to 1e-4.
+            priorc2 (float, optional): Prior on variant being causal for trait 2. Defaults to 1e-4.
+            priorc12 (float, optional): Prior on variant being causal for both traits. Defaults to 1e-5.
```
Comment (project-defiant) [inline src/gentropy/colocalisation.py:40]:
```suggestion
            priorc1 (float): Prior on variant being causal for trait 1. Defaults to 1e-4.
            priorc2 (float): Prior on variant being causal for trait 2. Defaults to 1e-4.
            priorc12 (float): Prior on variant being causal for both traits. Defaults to 1e-5.
```

File: src/gentropy/config.py [hunk 2]
```diff
@@ -38,6 +38,9 @@ class ColocalisationConfig(StepConfig):
     credible_set_path: str = MISSING
     coloc_path: str = MISSING
     colocalisation_method: str = MISSING
+    priorc1: float | None = MISSING
```
Comment (project-defiant) [inline src/gentropy/config.py:41]: It can be either Optional parameter (float | None) or required parameter `priroc1: float = MISSING`

**Review by project-defiant — COMMENTED** [review]
@addramir please see the commands. Does the parameters are required or optional?

File: src/gentropy/config.py (hunk 2 above)
Comment (addramir) [reply src/gentropy/config.py]: Done

File: src/gentropy/colocalisation.py (hunk 1 above)
Comment (addramir) [reply src/gentropy/colocalisation.py]: Done

**Review by project-defiant — APPROVED** [review]

**Comment by project-defiant** [comment]
Looks good, thanks for the update 👍👍

**Comment by d0choa** [comment]
If this is mandatory now... I need a set of priors for the orchestration configuration. Thoughts @addramir?

**Pull request merged**